app/styles/components/_category-filter.scss

```diff
@@ -9,13 +11,18 @@
   }
 
   .btn {
-    font-size: $font-size-medium;
-    padding: _get-rem(20px);
+    font-size: $font-size-regular;
+    padding-top: _get-rem(20px);
+    padding-bottom: 0;
+  }
+
+  .icon {
+    top: _get-rem(10px);
   }
 }
 
 .filter__cat-items {
-  padding-bottom: _get-rem(20px);
+  margin-top: _get-space('v', 'small');
```

Comment:
Vsetky zmeny v tomto subore su trochu na upratanie filtrov v kategorii nech to vyzera jednotnejsie ked je filter zbaleny/rozbaleny atd.